packages/skeleton/src/lib/utilities/Modal/Modal.svelte

```diff
@@ -115,6 +115,9 @@
 	};
 	let currentComponent: ModalComponent | undefined;
 	let registeredInteractionWithBackdrop = false;
+	let modalElement: HTMLDivElement;
+	let windowHeight: number;
+	let cBackdropOverflow: CssClasses = 'overflow-y-hidden';
```

Comment:
The `cX` classes are meant to be a fixed set of base classes that do not change. To avoid confusion, I'd rename and drop the "c". Likewise remove the CssClasses type designation - those are only needed for exposed props that users can override. It's an identifier that enables Tailwind Intellesense for those props. Given it's a privative, you can leave off the type entirely in fact!